### DISCUSSION TITLE
Autopilot speed limit

### DISCUSSION THREAD

**Discussion by MateusJK15** [FakeFishGames/Barotrauma#7947] [Ideas and suggestions]
Recently i made a simple "Silent Running" system and part of it is a simple logic that limits the maximum speed the sub can reach by deactivating a relay that feeds the engine when the submarine reaches a certain speed. But what if that was a native feature of the navigation terminal ? Using manual navigation you can fully control the submarine speed, but the autopilot always runs as fast as possible, preventing any stealth attempt. My suggestion is to add two buttons (or a slider) to the navigation terminal that can set the maximum speed the autopilot can reach.

**Comment by mygamingaccount** [comment]
There are many ways to limit speed, what you suggest is unnecessary.

**Comment by gerakis** [comment]
Try multiplying the speed out of the terminal by 0.1. This is how I do stealth and its bulletproof.
